webrender/src/resource_cache.rs

```diff
@@ -74,10 +74,12 @@ impl RenderedGlyphKey {
                size: Au,
                color: ColorF,
                index: u32,
+               x: f32,
```

Comment:
Should we change GlyphInstance to use those as well? Since we're basically passing data from there and GlyphInstance is just f32.